src/app/pages/home/home.component.html

```diff
@@ -29,4 +30,12 @@
       </p>
     </mat-card-content>
   </mat-card>
+  <mat-card class="privacy-card">
+    <mat-card-content class="privacy-card">
+      <button class="privacy-button" routerLink="/privacy-policy">
+        <mat-icon>privacy_tip</mat-icon>
+      </button>
+    </mat-card-content>
```

Comment:
This button looks really odd since it isn't a Material Button. 
![image](https://user-images.githubusercontent.com/1300395/94550534-7d9fe180-0219-11eb-94f6-8429fb8bfff4.png)

Also we probably shouldn't use only an icon because it's not very obvious what it means. I think we can get away without having the privacy policy or TOS on the home page since we will link to it on the about page but if we do want it on the home page we should probably just use a couple of simple links at the bottom of the page (which can replace the current ENRTF disclaimer card since that's now in the about page).

Reply:
We are going to move this to markdown, so this will not be an issue.